lr-cli/cortx_setup/network.py

```diff
@@ -0,0 +1,78 @@
+import argparse
+import logging
+
+from provisioner.commands import PillarSet
+from provisioner.salt import local_minion_id
+
+logger = logging.getLogger(__name__)
+
+
+class Network():
+
+    """Configure network"""
+
+    def config(self, transport_type=None, interface_type=None,
+            network_type=None, interfaces=None
+    ):
+        """Update provided network details in pillar data"""
```

Comment:
Provide sample values for the parameters in doc string.

Reply:
Addressed